CMakeLists.txt

```diff
@@ -608,6 +608,8 @@ IF (SONAME)
 	IF (LIBGIT2_FILENAME)
```

Comment:
yeah, nevermind, [sets the base name](https://cmake.org/cmake/help/v3.0/prop_tgt/OUTPUT_NAME.html) - fair enough